scripts/delete_failing_smt2_solver_tests

```diff
@@ -211,107 +131,57 @@ rm equality_through_array_of_struct1/test.desc
 rm equality_through_array_of_struct2/test.desc
 rm equality_through_array_of_struct3/test.desc
 rm equality_through_array_of_struct4/test.desc
-rm equality_through_struct1/test.desc
-rm equality_through_struct2/test.desc
-rm equality_through_struct3/test.desc
-rm equality_through_struct4/test.desc
-rm equality_through_struct5/test.desc
 rm equality_through_struct_containing_arrays1/test.desc
 rm equality_through_struct_containing_arrays2/test.desc
 rm equality_through_struct_containing_arrays3/test.desc
 rm equality_through_union1/test.desc
 rm equality_through_union2/test.desc
 rm equality_through_union3/test.desc
-rm exit1/test.desc
-rm extern_initialization1/test.desc
 rm fgets1/test.desc
-rm for2/test.desc
 rm full_slice1/test.desc
 rm full_slice2/test.desc
 rm gcc_bswap1/test.desc
-rm gcc_c99-bool-1/test.desc
-rm gcc_local_label1/test.desc
-rm gcc_popcount1/test.desc
 rm gcc_statement_expression4/test.desc
 rm gcc_switch_case_range1/test.desc
 rm gcc_switch_case_range2/test.desc
 rm gcc_vector1/test.desc
 rm gcc_vector2/test.desc
-rm getenv-overflow1/test.desc
-rm goto1/test.desc
-rm goto2/test.desc
-rm goto4/test.desc
-rm goto5/test.desc
 rm graphml_witness1/test.desc
 rm havoc_object1/test.desc
 rm hex_trace/test.desc
-rm if1/test.desc
 rm if2/test.desc
-rm if3/test.desc
-rm if4/test.desc
 rm inet_endian1/test.desc
-rm int-to-float2/test.desc
 rm integer-assignments1/test.desc
-rm json1/test.desc
 rm little-endian-array1/test.desc
-rm member1/test.desc
-rm memcpy1/test.desc
-rm memcpy2/test.desc
-rm memcpy3/test.desc
 rm memory_allocation1/test.desc
-rm memset1/test.desc
 rm memset3/test.desc
 rm mm_io1/test.desc
 rm no_nondet_static/test.desc
-rm null1/test.desc
-rm null2/test.desc
-rm path-per-path-vccs/test.desc
 rm pipe1/test.desc
-rm pointer-extra-checks/test.desc
 rm pointer-function-parameters/test.desc
 rm pointer-function-parameters-2/test.desc
-rm posix_memalign/test.desc
-rm printf1/test.desc
 rm read1/test.desc
 rm realloc1/test.desc
 rm realloc2/test.desc
-rm return2/test.desc
-rm return5/test.desc
 rm scanf1/test.desc
-rm self_loops_to_assumptions1/no-assume.desc
 rm simple_assert/test.desc
 rm stack-trace/test.desc
 rm strcat1/test.desc
-rm struct1/test.desc
 rm struct10/test.desc
-rm struct3/test.desc
 rm struct6/test.desc
 rm struct7/test.desc
-rm struct8/test.desc
 rm struct9/test.desc
-rm switch1/test.desc
-rm switch2/test.desc
-rm switch3/test.desc
-rm switch4/test.desc
-rm switch5/test.desc
-rm trace_address_arithmetic1/test.desc
 rm trace-values/trace-values.desc
-rm trace_options_json_extended/extended.desc
-rm trace_options_json_extended/non-extended.desc
-rm trace_show_code/test.desc
+rm trace_address_arithmetic1/test.desc
```

Comment:
Does this mean that ```trace_address_arithmetic1``` used to pass but no longer does?

Reply:
No, just that the list wasn't properly sorted before (scroll up a bit).

Reply:
Point.